### ISSUE TITLE
How to submit homeworks/problem sets

### ISSUE THREAD

**Issue by JackShiqiLi** [OpenSourceEcon/BootCamp2019#12]
Hi everyone,
I thought Rick has talked about how to submit homeworks but I forgot it, do we just push them to our repos?
@rickecon 
Many Thanks.

**Comment by rebekahanne** [comment]
Yes, you should push them to your forks of the BootCamp2019 repository, but according to the following file structure: You should create a ProblemSets folder in the main directory of your fork, create subfolders for Computation and Econ, and create subfolders within each of these for each problem set.

For example, you would put your week 1 computation problem set in: BootCamp2019/ProblemSets/Computation/PS1

(@rickecon Is this file structure ok?)

**Comment by rickecon** [comment]
Yes. @rebekahanne is right. You will push your computation problem sets (code, PDF, and Jupyter notebooks) to your fork of the main repo. For the computational problem sets [PS1a](https://github.com/OpenSourceEcon/BootCamp2019/blob/master/Computation/Wk1_DifInt/DifInt_probset1a.pdf) and [PS1b](https://github.com/OpenSourceEcon/BootCamp2019/blob/master/Computation/Wk1_DifInt/DifInt_probset1b.pdf), you should push those solutions to the following folder:
```
/YourHandle/BootCamp2019/ProblemSets/Computation/PS1
```
For the dynamic programming problem set, push your code and/or PDF and/or Jupyter notebook to:
```
/YourHandle/BootCamp2019/ProblemSets/Econ/PS1
```
All of your problem sets will either be in the `Econ` folder or in the `Computation` folder.

@ani1231091 @AOzhegova @ashenk99 @azinoma @danielkdimitrov @EricHuang49 @isaacsantelli @JackShiqiLi @janertl @jeannesorin @joaopvalente @keertanavc @keiirizawa @kumar-ab @liu431 @lucamaria @MartinaFraschini @martinlibo @mathyoufitzgerald @ozaltun @rebekahanne @rmmomin @sofoniasalemu @terrytwu @tpellet @uta-bolt @Wouter-vdW @Xincheng-Qiu
